lib/pages/loginPages/CreateAccountPage.dart

```diff
@@ -50,4 +107,67 @@ class _CreateAccountPageState extends State<CreateAccountPage> {
         ),
       );
     }
-}
\ No newline at end of file
+    void createAccount() async {
+
+      _onLoading();
+
+      if (passwordConfirmController.text != passwordController.text) {
+        displayError("The passwords do not match", "Re-enter passwords", "OK");
+      } else if (emailController.text.isEmpty || passwordController.text.isEmpty || passwordConfirmController.text.isEmpty){
+        displayError("Missing email or password","Please provide both an email and password", "OK");
+      } else {
+       await this.widget.auth.createUserWithEmailAndPassword(email: emailController.text, password: passwordController.text).catchError((error) {
+         displayError("Account Creation Failed!","Sorry, the account could not be created.", "OK");
```

Comment:
https://github.com/flutter/flutter/issues/20223

Did a quick google search, and it looks like there isn't a good way currently to access what the error is specifically. Will do a bit more digging.

Reply:
Okay, this is not a high priority. For now I would say to change the error message to include a short blurb about how the email may already be in use or whatnot. Most of the time this error will occur because email is in use but not always.